camel/interpreters/subprocess_interpreter.py

```diff
@@ -98,15 +98,70 @@ def run_file(
         if not file.is_file():
             raise RuntimeError(f"{file} is not a file.")
         code_type = self._check_code_type(code_type)
-        cmd = shlex.split(
-            self._CODE_EXECUTE_CMD_MAPPING[code_type].format(
-                file_name=str(file)
+        if code_type == "python":
```

Comment:
Should we do `if _CODE_TYPE_MAPPING[code_type] == "python":` to support `code_type` for `py3`, `python3`, `py` as well?